example/lambdas/db-drop-tables/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist/lambda",
+    "declaration": false,
+    "declarationMap": false,
+    "sourceMap": false,
```

Comment:
We don't care about source-maps because the Lambda is so small?

Reply:
That was my feeling, but I can re-enable them if you want. This lambda is only being used in CI to drop tables.